### ISSUE TITLE
Does not render isomorphically

### ISSUE THREAD

**Issue by jamesjjk** [davidtheclark/react-aria-tabpanel#8]
Hi nice build, but it does not render isomorphically.. the document is not available when rendering serverside. CreateManager is called on componentWillMount which calls focus-group. Consider using componentDidMount which is called client side only.

https://facebook.github.io/react/docs/component-specs.html

```
react-aria-tabpanel/node_modules/focus-group/index.js:18
  document.addEventListener('keydown', this._boundHandleKeydownEvent, true);
  ^

ReferenceError: document is not defined
```

**Comment by davidtheclark** [comment]
Open to a PR. I am not personally too interested in dealing with the complexities of isomorphism for the questionable gains (no gains, really, for the app that I work on). But sounds like an easy fix!

**Comment by jamesjjk** [comment]
@davidtheclark Hi David, will do a PR. Its a straightforward fix, need to call activate of focusgroup in componentdidmount lifecycle method. It would seem a shame that the only aria tabs implementation would not work isomorphically..

**Comment by jamesjjk** [comment]
#9

**Comment by davidtheclark** [comment]
Fixed by #9.
